recipe/bld.bat

```diff
@@ -1,4 +1,9 @@
-%PYTHON% build/build.py
+set BAZEL_VS="C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools"
+set BAZEL_VC="C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/VC"
+set CLANG_COMPILER_PATH=%BUILD_PREFIX:\=/%/Library/bin/clang.exe
+set BAZEL_LLVM=%BUILD_PREFIX:\=/%/Library/
+
```

Comment:
If you add another commit, can you reference this link here too?
https://github.com/openxla/xla/blob/edf18ce242f234fbd20d1fbf4e9c96dfa5be2847/.bazelrc#L383